src/config/dynamic-theme-fixes.config

```diff
@@ -2362,6 +2362,14 @@ th {
 
 ================================
 
+bugs.python.org
+
+INVERT
+img[title="has PR"]
+img[title="GitHub"]
```

Comment:
Should this rather be title="has patch"?

Reply:
Yes... that should also be inverted(barely visible). This title="Github" doesn't seem to match any icons.

Reply:
title="Github" is at the left side of each [issue page](https://bugs.python.org/issue40255):
<img width="885" alt="Screen Shot 2022-02-22 at 16 04 56" src="https://user-images.githubusercontent.com/21101839/155088873-9fd54b62-16f3-437f-9fd2-2d81a9a7034d.png">

Invert the clip icon doesn't have any effect.